ci/pipeline.yml

```diff
@@ -44,7 +44,6 @@ jobs:
       inputs:
       - name: pull-request
       params:
-        TERRAFORM_BIN: terratest-14.11
```

Comment:
Does this `params` section need to stay now that there's nothing in it?

Reply:
Better to leave it there in case we need it again